.github/workflows/docs_bump_detected.yml

```diff
@@ -0,0 +1,63 @@
+name: Docs Bump Detected
+
+on:
+  pull_request:
+    paths:
+      - 'docs/**'
+
+jobs:
+  push_workflow_to_docs_metabase_github_io_job:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Print changed docs files
+        run: |
+          echo "A change in the docs directory was detected."
+
+      - name: Prepare to trigger slurp_branches workflow in metabase/docs.metabase.github.io
+        run: |
+          if [ "${{ github.event_name }}" = "pull_request" ]; then
+            echo "BRANCH_NAME=${{ github.head_ref }}" >> $GITHUB_ENV
+          else
+            echo "BRANCH_NAME=${GITHUB_REF#refs/heads/}" >> $GITHUB_ENV
+          fi
+
```

Comment:
I think you can just at a `repository_dispatch` event to the other repo workflow and call it like this, no need to go searching for its id

```
   - uses: actions/github-script@v7
        with:
          github-token: ${{ secrets.METABASE_AUTOMATION_USER_TOKEN }}
          script: | # js
            github.rest.repos.createDispatchEvent({
              owner: '${{ github.repository_owner }}',
              repo: 'metabase.github.io',
              event_type: 'process_docs_changes.yml',
              client_payload: {
                branch: '${{ process.env.BRANCH_NAME }}',
              }
            });
```

Reply:
I'll give that a try, thanks

Reply:
That worked well, thanks Ryan